gateway-ha/src/main/java/io/trino/gateway/ha/module/HaGatewayProviderModule.java

```diff
@@ -81,6 +82,11 @@
 public class HaGatewayProviderModule
         extends AbstractModule
 {
+    private static final String MONITOR_HTTP_CLIENT_KEY_STORE_PATH = "monitor.http-client.key-store-path";
+    private static final String MONITOR_HTTP_CLIENT_KEY_STORE_PASSWORD = "monitor.http-client.key-store-password";
+    private static final String MONITOR_HTTP_CLIENT_TRUST_STORE_PATH = "monitor.http-client.trust-store-path";
+    private static final String MONITOR_HTTP_CLIENT_TRUST_STORE_PASSWORD = "monitor.http-client.trust-store-password";
+
```

Comment:
Actually I was thinking of something like this, WDYT?

```suggestion
    private static final List<String> REQUIRED_MONITOR_MTLS_KEYS = List.of(
            "monitor.http-client.key-store-path",
            "monitor.http-client.key-store-password",
            "monitor.http-client.trust-store-path",
            "monitor.http-client.trust-store-password");
```